modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -472,8 +501,12 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
     save(site.locs, t, FORECAST, ANALYSIS, enkf.params, new.state, new.params,
          out.configs, ensemble.samples, inputs, Viz.output,
          file = file.path(settings$outdir,"SDA", "sda.output.Rdata"))
-    #writing down the image - either you asked for it or nor :)
-    if (t%%2==0 | t==nt)  post.analysis.multisite.ggplot(settings, t, obs.times, obs.mean, obs.cov, FORECAST, ANALYSIS ,plot.title=control$plot.title, facetg=control$facet.plots, readsFF=readsFF)
+    #writing down the image - either you asked for it or not :)
+
+#   if (t%%2==0 | t==nt)  post.analysis.multisite.ggplot(settings, t, obs.times, obs.mean, obs.cov, FORECAST, ANALYSIS ,plot.title=control$plot.title, facetg=control$facet.plots, readsFF=readsFF)
+    if (t == 1){
+      unlink(list.files(outdir, "*.nc", recursive = TRUE, full.names = TRUE))
```

Comment:
You don't need the `if`. You can move this to line 255.

Reply:
That originally had to do with removing the .nc files that were created for all years at the beginning of the run, but keep the other .nc files that are outputted. I added this to the "keepNC" option that I added so it's gone now